frontend/src/pages/Afterword/index.js

```diff
@@ -102,51 +120,49 @@ class Afterword extends React.Component<Props, State> {
         <ContentContainer>
-          <Title>Thanks for taking the exercise.</Title>
-          {this.props.afterwordMessage && (
-            <MarkdownContainer>
-              <ReactMarkdown source={this.props.afterwordMessage} />
-            </MarkdownContainer>
+          {afterwordMessage && (
+            <>
+              <DebriefTitle>Debrief</DebriefTitle>
+              <Description>{afterwordMessage}</Description>
+            </>
+          )}
+          {scores && (
+            <List>
+              <StructuredListHead>
+                <ListRow head>
+                  {this.getHeaders().map((header, index) => (
+                    <ListCell key={`${index}-${header.header}`} head>
+                      {header.header}
+                    </ListCell>
+                  ))}
+                </ListRow>
+              </StructuredListHead>
+              <ListBody>
+                {scores.map((row, index) => (
+                  <ListRow key={`${index}-${row.task}`}>
+                    <ListCell>{row.task}</ListCell>
+                    <ListCell>{row.score}</ListCell>
+                    <ListCell>{row.debrief}</ListCell>
+                  </ListRow>
+                ))}
+              </ListBody>
+            </List>
           )}
-          {this.state.scores && (
-            <DataTable
-              rows={this.state.scores}
-              headers={this.getHeaders()}
-              render={({ rows, headers, getHeaderProps }) => {
-                return (
-                  <TableContainer title="Debrief">
-                    <Table>
-                      <TableHead>
-                        <TableRow>
-                          {headers.map((header, index) => (
-                            <TableHeader
-                              // eslint-disable-next-line react/no-array-index-key
-                              key={`${index}-${header.header}`}
-                              {...getHeaderProps({ header })}
-                            >
-                              {header.header}
-                            </TableHeader>
-                          ))}
-                        </TableRow>
-                      </TableHead>
-                      <TableBody>
-                        {rows.map(row => (
-                          // eslint-disable-next-line react/no-array-index-key
-                          <TableRow key={row.id}>
-                            {row.cells.map(cell => (
-                              <TableCell key={cell.id}>{cell.value}</TableCell>
-                            ))}
-                          </TableRow>
-                        ))}
-                      </TableBody>
-                    </Table>
-                  </TableContainer>
-                );
-              }}
-            />
+          {this.state.debrief && (
+            <Button
+              onClick={() =>
+                this.props.history.push(`/debrief/${participantId}`)
```

Comment:
`participantId` is not part of props so this is undefined. You will need to use react-router to get this. you have already added `withRouter` (which is what we need) So all you need to do now is get the id from the url.

`this.props.match.params.participantUuid`